### PR TITLE
tools: support to request the Local TSO from a specific DC

### PR DESCRIPTION
**Pull request by JmPotato**
Signed-off-by: JmPotato <ghzpotato@gmail.com>

### What problem does this PR solve?

Support to request the Local TSO from a specific DC.

### What is changed and how it works?

* Fix a typo.
* Add a new flag to specify the `dc-location` for `GetLocalTSO`.

### Check List

Tests

- Manual test (add detailed scripts or steps below)

### CODE REVIEW

**Review by nolouch — COMMENTED** [review]
LGTM

**Comment by JmPotato** [comment]
> LGTM

We should use `/lgtm` now, lol.

File: tools/pd-tso-bench/main.go [hunk 1]
```diff
@@ -99,7 +100,7 @@ func bench(mainCtx context.Context) {
 	ctx, cancel := context.WithCancel(mainCtx)
 	// To avoid the first time high latency.
 	for idx, pdCli := range pdClients {
-		_, _, err := pdCli.GetTS(ctx)
```
Comment (rleungx) [inline tools/pd-tso-bench/main.go:102]: It seems we don't need GetTS anymore?

Comment (JmPotato) [reply tools/pd-tso-bench/main.go:102]: Yes, it's equivalent to getting a local TSO from the "global". But considering the backward compatibility, maybe we should keep it in the PD client?

Comment (rleungx) [reply tools/pd-tso-bench/main.go:102]: How about marking it as deprecated?

**Review by rleungx — APPROVED** [review]
Looks good. BTW, it's better to open an issue to record the GetTS and GetLocalTS problem

**Comment by JmPotato** [comment]
> Looks good. BTW, it's better to open an issue to record the GetTS and GetLocalTS problem

Here it comes #3165.

**Review by Yisaer — COMMENTED** [review]
/lgtm

**Review by Yisaer — APPROVED** [review]
LGTM

**Pull request merged**